srml/balances/src/lib.rs

```diff
@@ -175,6 +176,9 @@ pub trait Subtrait<I: Instance = DefaultInstance>: system::Trait {
 	type Balance: Parameter + Member + SimpleArithmetic + Codec + Default + Copy +
 		MaybeSerializeDebug + From<Self::BlockNumber>;
 
+	/// Handler for converting transactions weights to fees
```

Comment:
in what context does this happen/is this used?

Reply:
the `make_payment` function (line 1048)